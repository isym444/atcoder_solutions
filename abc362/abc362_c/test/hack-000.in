10
-4 8
15 16
-2 -1
17 18
-4 8
7 20
-1 14
-8 10
-1 -1
-19 -19
