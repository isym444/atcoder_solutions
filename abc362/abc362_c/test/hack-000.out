Yes
-4 15 -2 17 -4 7 -1 -8 -1 -19 
